paddle/fluid/framework/new_executor/stream_analyzer.cc

```diff
@@ -18,6 +18,12 @@
 namespace paddle {
 namespace framework {
 
+bool StreamAnalyzer::InAsyncStream(const OpFuncNode& op_func_node) {
```

Comment:
I think the better conditon is: `stream of op_node != stream of input tensor`

Reply:
Same above.